docs/apache-airflow/core-concepts/params.rst

```diff
@@ -219,6 +277,9 @@ For examples also please take a look to two example DAGs provided: ``example_par
 
 .. image:: ../img/trigger-dag-tutorial-form.png
 
+The trigger form can also be forced to be displayed also if no params are defined using the configuration switch
+``webserver.show_trigger_form_if_no_params``.
```

Comment:
This is 2.7.0 only doesn't it? 
lets use
`.. versionadded:: 2.7.0`

Reply:
yeah. the comment should be under version_added.

Reply:
Adjusted,